### DISCUSSION TITLE
.sln doesn't generate all files, such as PhysX3ExtensionsDEBUG.lib

### DISCUSSION THREAD

**Discussion by TheRickyZhang** [NVIDIA-Omniverse/PhysX#293] [Q&A]
I am having a similar issue as [here](https://forums.developer.nvidia.com/t/problem-with-pxdefaulterrorcallback-physx-3-4/59304/3) where compiling my PhysX.sln doesn't build things like PhysX3Extensions.lib or PhysXExtensions_64.dll. I am using VS2022 - any help would be appreciated!

**Comment by vreutskyy** [comment]
Hi @TheRickyZhang 
I just tried it. For me it generates PhysXExtensions_static_64.lib (in physx\bin\win.x86_64.vc143.mt\<config> folder) and no DLL as it's static.
